### PR TITLE
ZEPPELIN-3635. Drop support for spark before 1.6

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?
This PR is to remove legacy code to drop support for spark before 1.6

### What type of PR is it?
[Refactoring]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-3635

### How should this be tested?
* Ci pass

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: spark/interpreter/src/main/java/org/apache/zeppelin/spark/SparkVersion.java [hunk 1]
```diff
@@ -86,37 +80,14 @@ public static SparkVersion fromVersionString(String versionString) {
...
-
-  public boolean oldLoadFilesMethodName() {
-    return this.olderThan(SPARK_1_3_0);
-  }
-
-  public boolean oldSqlContextImplicits() {
-    return this.olderThan(SPARK_1_3_0);
-  }
-
   public boolean isSpark2() {
```
Comment (felixcheung) [inline spark/interpreter/src/main/java/org/apache/zeppelin/spark/SparkVersion.java]: should we check if older than 1.6 and fail?

**Comment by felixcheung** [comment]
looks like we don't test older spark versions..

**Comment by jongyoul** [comment]
BTW, we could also remove profiles for spark 1.4 - 1.5

**Comment by zjffdu** [comment]
Thanks @jongyoul  I have removed these profiles

**Pull request closed without merging**